GitUI/Script/ScriptOptionsParser.cs

```diff
@@ -426,34 +428,34 @@ private static string GetRemotePath(string url)
                     break;
 
                 case "cDefaultRemoteUrl":
-                    if (string.IsNullOrEmpty(currentRemote))
+                    if (string.IsNullOrEmpty(currentRemote) || uiCommands.GitModule is null)
```

Comment:
Remove unnecessary `using` and `null` checks: ab70b8e974054ce100e0c81642ecf606f083c411